Dockerfile.build-tools

```diff
@@ -72,6 +72,18 @@ RUN curl -fsSL 'https://apt.llvm.org/llvm-snapshot.gpg.key' | apt-key add - \
     && bash -c 'for f in /usr/bin/clang*-${LLVM_VERSION} /usr/bin/llvm*-${LLVM_VERSION}; do ln -s "${f}" "${f%-${LLVM_VERSION}}"; done' \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Install docker
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
```

Comment:
Why don't we use 
```
> /etc/apt/sources.list.d/docker.list 
```
instead of 
```
| tee /etc/apt/sources.list.d/docker.list > /dev/null
```
and shouldn't we use 
```
${VERSION_CODENAME} 
```
for consistency?

Reply:
I believe I've copied this snippet from the Docker website. I'm not sure why they use `| tee ... > /dev/null`.
Replaced  with `> ...` and with `${DEBIAN_VERSION_CODENAME}`  (a new arg to not to use instead of `$(. /etc/os-release && echo "$VERSION_CODENAME")` or hardcoded `bullseye` string)